docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -96,6 +98,34 @@ This video provides a walkthrough of revoking installations, covering the key id
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/MIw9x1Z4WXw?si=4bcbfkHTDvsM0uDG" title="YouTube video player" frameBorder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerPolicy="strict-origin-when-cross-origin" allowFullScreen></iframe>
 
+### Revoke an installation
+
+You can revoke a specific installation other than the currently accessed installation. You can use this function to revoke an installation when an inbox ID has reached its maximum of 5 installations.
```

Comment:
Might be worth adding a note that your account may have more than 5 installations already in that case an error will be thrown that will tell you how many installations you have example 20 and that you will need to revoke 16 installations to be able to add 1 more.

Reply:
@nplasterer When will this error display -- when the app updates to use this SDK patch version and the user opens the app? Or when the user tries to add the 21st installation?

Reply:
only when they try to add a new installation